### PR TITLE
qcloud and googlecloud upload file

### PR DESCRIPTION
**Pull request by smilemusic**
add qcloud and googlecloud upload file

```
                                                       ---yangshuai
```

### CODE REVIEW

**Comment by MabinGo** [comment]
Please resolve the conflicts and update your branch.Then update the PR by the same git log.

File: backend/backend.go [hunk 1]
```diff
@@ -10,6 +10,8 @@ import (
 	_ "github.com/containerops/dockyard/backend/drivers/oss"
 	_ "github.com/containerops/dockyard/backend/drivers/qiniu"
 	_ "github.com/containerops/dockyard/backend/drivers/upyun"
+	_ "github.com/containerops/dockyard/backend/drivers/qcloud"
+	_ "github.com/containerops/dockyard/backend/drivers/googlecloud"
```
Comment (MabinGo) [inline backend/backend.go:14]: please excute gofmt command to format your codes after they are updated

File: backend/drivers/googlecloud/googlecloud.go [hunk 2]
```diff
@@ -23,53 +25,30 @@ var (
...
-		log.Fatalf("GCS reading conf/runtime.conf, GCS get privateKey err, is nil")
-	}
-
-	var privateKeyFile string
-	if privateKeyFile = conf.String("Googlecloud::privatekey"); privateKeyFile == "" {
-		log.Fatalf("GCS reading conf/runtime.conf, GCS get privateKey err, is nil")
-	}
-	privateKey, err = ioutil.ReadFile(gopath + "/src/github.com/containerops/dockyard/conf/" + privateKeyFile)
+	fmt.Println("PrivateKeyFile=", setting.PrivateKeyFile)
+	privateKey, err = ioutil.ReadFile(gopath + "/src/github.com/containerops/dockyard/conf/" + setting.PrivateKeyFile)
```
Comment (MabinGo) [inline backend/drivers/googlecloud/googlecloud.go:43]: It is not permitted to use hard coding

File: backend/drivers/googlecloud/googlecloud.go [hunk 3]
```diff
@@ -23,53 +25,30 @@ var (
...
-		log.Fatalf("GCS reading conf/runtime.conf, GCS get clientEmail err, is nil")
-	}
-
-	InjectReflect.Bind("googlecloudsave", googlecloudsave)
-}
-
-func googlecloudsave(file string) (url string, err error) {
-
-	s := []string{scope}
+	fmt.Println("mytest1.....")
```
Comment (MabinGo) [inline backend/drivers/googlecloud/googlecloud.go:47]: debug info that is added during development must be deleted

File: backend/drivers/qcloud/qcloud.go [hunk 4]
```diff
@@ -6,13 +6,17 @@ import (
...
 	"fmt"
 	"net/http"
-	"net/url"
 	"os"
-	"sort"
 	"strings"
 	"time"
-
-	"github.com/astaxie/beego/config"
+	"bytes"
```
Comment (MabinGo) [inline backend/drivers/qcloud/qcloud.go:12]: please adjust the import packages in order to follow Dockyard programming spec

File: backend/drivers/qcloud/qcloud.go [hunk 5]
```diff
@@ -105,46 +83,91 @@ func qcloudsave(file string) (url string, err error) {
 	if err != nil {
 		return "", err
 	}
-	filesize := fi.Size()
-
+//	filesize := fi.Size()
```
Comment (MabinGo) [inline backend/drivers/qcloud/qcloud.go:86]: please delete the redundant codes and personal debug info

File: backend/drivers/qcloud/qcloud.go [hunk 6]
```diff
@@ -105,46 +83,91 @@ func qcloudsave(file string) (url string, err error) {
...
+	rsp, err := client.Do(req)
 	if err != nil {
 		return "", err
+	}else{
+		fmt.Println("qcloudsave---StatusCode=", rsp.StatusCode)
+	}
+	for i, k := range rsp.Header {
+		fmt.Printf("rsp.Header--------%v = %v\n", i, k)
 	}
+	testfile, _ := os.Create("test_body.txt")
```
Comment (MabinGo) [inline backend/drivers/qcloud/qcloud.go:165]: please delete the personal debug info

File: backend/drivers/qcloud/qcloud.go [hunk 7]
```diff
@@ -152,16 +175,16 @@ func qcloudGetDownloadUrl(fileName string) (downloadUrl string) {
...
+	params["bucketId"] = setting.QcloudBucket
+	params["secretId"] = setting.QcloudAccessKeyID
 	params["path"] = "/" + fileName
 	time := fmt.Sprintf("%v", time.Now().Unix())
 	params["time"] = time
 
 	downloadPlainText := makePlainText("", params)
 	sign := sign(downloadPlainText, QcloudAccessKeySecret)
-	url := "cos.myqcloud.com/" + QcloudAccessID + "/" + QcloudBucket + "/" + fileName + "?" + "secretId=" + QcloudAccessKeyID + "&time=" + time
+	url := "http://" + "test-10014152.file.myqcloud.com/" + setting.QcloudAccessID + "/" + setting.QcloudBucket + "/" + fileName + "?" + "secretId=" + setting.QcloudAccessKeyID + "&time=" + time
```
Comment (MabinGo) [inline backend/drivers/qcloud/qcloud.go:187]: It is not permitted to use hard coding

File: conf/googlecloud.key [hunk 8]
```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```
Comment (MabinGo) [inline conf/googlecloud.key:1]: Can not add a personal key as a default setting

File: handler/acidiscover.go [hunk 9]
```diff
@@ -16,6 +16,7 @@ func DiscoveryACIHandler(ctx *macaron.Context, log *logs.BeeLogger) {
 	namespace := ctx.Params(":namespace")
 	aciname := ctx.Params(":aciname")
 
+	fmt.Println("DiscoveryACIHandler-------------------------------")
```
Comment (MabinGo) [inline handler/acidiscover.go:19]: please delete all the personal debug info,the following is the same

**Comment by smilemusic** [comment]
this request is not used

**Pull request closed without merging**